plugins/mac-virtualcam/src/obs-plugin/plugin-main.mm

```diff
@@ -68,6 +68,7 @@ - (OSSystemExtensionReplacementAction)request:(nonnull OSSystemExtensionRequest
                   actionForReplacingExtension:(nonnull OSSystemExtensionProperties *)existing
                                 withExtension:(nonnull OSSystemExtensionProperties *)ext
 {
+    blog(LOG_INFO, "mac-camera-extension: Replacement requested, replacing existing extension.");
```

Comment:
I'd rather have the version information logged as well, that way we can tell whether the replacement is a downgrade vs an upgrade (as we allow both, we also need to check for both).

Reply:
Ah yes, good idea

Reply:
Looks like this was addressed?

Reply:
Yep.